packages/config-react-webpack/config/webpack.config.umd.js

```diff
@@ -30,10 +38,10 @@ module.exports = () => {
 						test: useTypescript ? /\.(js|ts|tsx)$/ : /\.js$/,
 						exclude: /node_modules/,
 						use: [
-							{ loader: 'cache-loader' },
+							!process.env.NO_CACHE_LOADER && { loader: 'cache-loader' },
```

Comment:
Can we use --env.nocache instead to pass options to webpack ?
```suggestion
							!env.nocache && { loader: 'cache-loader' },
```